### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by KingsleyIbe**
In this milestone, I accomplished the following:

- Applied Gitflow workflow method
- Created project using an npm webpack
- Wrote a plain HTML markup with minimum styling 
- Used ES6 modules, with import and export.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
       },
     ],
   },
-  
+
```
Comment (rloterh) [inline webpack.config.js:25]: - [ ] To maintain a standard project setup, kindly ensure that **all** the files within the `webpack-demo` folder **reside in the root directory** of your project

**Comment by KingsleyIbe** [comment]
Hello Robert, 

Thank you for reviewing my project. I will implement the changes.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @KingsleyIbe , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

tenor (1)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hello Brain,

Thank you for your prompt feedback

**Pull request merged**